compiler/src/main/scala/org/scalajs/core/compiler/JSTreeExtractors.scala

```diff
@@ -14,23 +14,51 @@ import org.scalajs.core.ir.Types._
 object JSTreeExtractors {
 
   object jse {
+
     /**
-     *  A literally named sequence (like in a call to applyDynamicNamed)
+     *  A sequence of tuples (like in a call to applyDynamicNamed)
      *
      *  Example (Scala): method(("name1", x), ("name2", y))
      */
-    object LitNamed {
-      def unapply(exprs: List[Tree]) = unapply0(exprs, Nil)
-
+    private[jse] trait GenLitNamed {
```

Comment:
Actually, I meant it should be a `private object`. It really doesn't need to be a superclass/trait of `LitNamed` and `ExtractLitNamed`.